sizereport.config.js

```diff
@@ -1,6 +0,0 @@
-module.exports = {
```

Comment:
This change is unrelated to the PR. This file would configure the [size report](https://github.com/GoogleChromeLabs/travis-size-report) for Travis, which we no longer use and so this file is no longer needed.